### PR TITLE
Update redshift-configs.md

### PR DESCRIPTION
**Pull request by amychen1776**
## What are you changing in this pull request and why?
Fix the auto_refresh field. This has been bugging me for far too long

### CODE REVIEW

File: website/docs/reference/resource-configs/redshift-configs.md [hunk 1]
```diff
@@ -109,9 +109,11 @@ models:
 The Redshift adapter supports [materialized views](https://docs.aws.amazon.com/redshift/latest/dg/materialized-view-overview.html) and refreshes them for every subsequent `dbt run` that you execute. For more information, see [Refresh Materialized Views](https://docs.aws.amazon.com/redshift/latest/dg/materialized-view-refresh.html) in the Redshift docs.
 
 Materialized views support the optional configuration `on_configuration_change` with the following values: 
-- `apply` (default) &mdash; attempts to update the existing database object if possible, avoiding a complete rebuild. The `auto_refresh` action can applied without the need to rebuild the materialized view.
+- `apply` (default) &mdash; attempts to update the existing database object if possible, avoiding a complete rebuild. 
 - `skip` &mdash; allows runs to continue while also providing a warning that the model was skipped
```
Comment (mikealfare) [inline website/docs/reference/resource-configs/redshift-configs.md]: Can we also correct this to `continue` while we're at it?

Comment (amychen1776) [reply website/docs/reference/resource-configs/redshift-configs.md]: of course

Comment (amychen1776) [inline website/docs/reference/resource-configs/redshift-configs.md]:
```suggestion
- `continue` &mdash; allows runs to continue while also providing a warning that the model was skipped
```

Comment (amychen1776) [inline website/docs/reference/resource-configs/redshift-configs.md]:
```suggestion
- `continue` &mdash; allows runs to continue while also providing a warning that the model was not executed
```

File: website/docs/reference/resource-configs/redshift-configs.md [hunk 2]
```diff
@@ -109,9 +109,11 @@ models:
 The Redshift adapter supports [materialized views](https://docs.aws.amazon.com/redshift/latest/dg/materialized-view-overview.html) and refreshes them for every subsequent `dbt run` that you execute. For more information, see [Refresh Materialized Views](https://docs.aws.amazon.com/redshift/latest/dg/materialized-view-refresh.html) in the Redshift docs.
 
 Materialized views support the optional configuration `on_configuration_change` with the following values: 
-- `apply` (default) &mdash; attempts to update the existing database object if possible, avoiding a complete rebuild. The `auto_refresh` action can applied without the need to rebuild the materialized view.
-- `skip` &mdash; allows runs to continue while also providing a warning that the model was skipped
-- `fail` &mdash; forces runs to fail if a change is detected in a materialized view 
+- `apply` (default) &mdash; attempts to update the existing database object if possible, avoiding a complete rebuild. 
+- `continue` &mdash; allows runs to continue while also providing a warning that the model was skipped
```
Comment (amychen1776) [inline website/docs/reference/resource-configs/redshift-configs.md]:
```suggestion
- `continue` &mdash; allows runs to continue while also providing a warning that the model was not executed
```

File: website/docs/reference/resource-configs/redshift-configs.md [hunk 3]
```diff
@@ -125,6 +127,7 @@ The following examples create a materialized view:
 {{
   config(
     materialized = 'materialized_view',
+    auto_refresh = false,
```
Comment (mikealfare) [inline website/docs/reference/resource-configs/redshift-configs.md]: I believe this is a capital F; it's not the jinja `false`, it's the python `False`.

We're also adding the option to pass in case-insensitive strings, e.g. `"False"`, `"false"`, `"FALSE"`, etc. It's not there yet, and we can submit another PR that goes in at the same time if that makes more sense.

Comment (amychen1776) [reply website/docs/reference/resource-configs/redshift-configs.md]: I hate that but glad that's getting fixed :)

Comment (amychen1776) [inline website/docs/reference/resource-configs/redshift-configs.md]:
```suggestion
    auto_refresh = False,
```

**Review by mikealfare — APPROVED** [review]
